src/Tribe/Admin/Settings.php

```diff
@@ -46,6 +46,18 @@ public function get_url( array $args = [] ) {
 
 		$wp_url = is_network_admin() ? network_admin_url( 'settings.php' ) : admin_url( 'edit.php' );
 
+		if ( $args['frag'] ?? false ) {
```

Comment:
One minor suggestion: To help make it more clear how this parameter is used, I think it would be good to use `anchor` as the argument key instead of `frag`, since we're using a URL with an anchor ID in it.